src/intro/no-std.md

```diff
@@ -1,5 +1,74 @@
 # A `no_std` Rust Environment
 
-> ❌: **This section should cover:**
->
-> * what #[no_std] means: no I/O (filesystems), allocation, etc.
\ No newline at end of file
+As mentioned earlier, this book is all about "bare metal" programming. Standard libraries in turn usually
+require some sort of system (os) integration and higher level functionality, on top of which they implement their functionality.
+Such functionalities are for example I/O (files, networking), Threads, and Heap (dynamic memory allocation).
+
+But don't worry, this does not mean you have to code everything from scratch.
+In an `no_std` environment you still will be able to use the [libcore](https://doc.rust-lang.org/core/),
+if you provide a few basic functions (Either by pulling in in the [rlibc](https://crates.io/crates/rlibc) crate,
```

Comment:
I didn't even know rlibc existed. ;) Looks pretty much dead to me. Very little is needed to compile a program for a MCU nowadays, mostly linker magic.